src/pages/docs/creators/ray/basic-ray-tasks-usage-tutorial.md

```diff
@@ -16,38 +16,59 @@ You can find the examples in [`ray-on-golem` repository](https://github.com/gole
 
 Have a look at the code (but you can find it also in [the repository](https://github.com/golemfactory/golem-ray/blob/main/examples/simple-task.py):
 ```python
-from collections import Counter
 import socket
 import time
+import argparse
+from collections import Counter
 
 import ray
 ray.init()
 
-print('''This cluster consists of
+
+# output cluster information before the computation
+print(
```

Comment:
couldn't we put this `print` into a separate function as the call seems identical?